NOTICE.txt

```diff
@@ -12173,41 +12173,6 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
 
----
```

Comment:
We will need to keep this as we are not removing the library

Reply:
@M-ZubairAhmed No, with this PR, `stylelint` would be only in `devDependencies` as explained in the PR description. See if we do not have `jest` in `NOTICE.txt`, for example, while it (`jest`) is (only) in `devDependencies`: https://github.com/mattermost/mattermost/blob/6d427cf005e4013f230d3f74c51840ce6cc99efd/webapp/channels/package.json#L159

Also you can see as follow:
```
$ cd webapp/channels; npm why stylelint
stylelint@16.2.1 dev
node_modules/stylelint
  stylelint@"^16.2.1" from @stylistic/stylelint-plugin@2.1.0
  node_modules/@stylistic/stylelint-plugin
    dev @stylistic/stylelint-plugin@"2.1.0" from mattermost-webapp@9.3.0
    channels
      mattermost-webapp@9.3.0
      node_modules/mattermost-webapp
        workspace channels from the root project
  dev stylelint@"16.2.1" from mattermost-webapp@9.3.0
  channels
    mattermost-webapp@9.3.0
    node_modules/mattermost-webapp
      workspace channels from the root project
```

Reply:
You are absolutely correct